core/contracts/oracle/implementation/Voting.sol

```diff
@@ -631,11 +629,7 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
      *    PRIVATE AND INTERNAL FUNCTIONS    *
      ****************************************/
 
-    function _getPriceOrError(bytes32 identifier, uint time)
-        private
-        view
-        returns (bool _hasPrice, int price, string memory err)
-    {
+    function _getPriceOrError(bytes32 identifier, uint time) private view returns (bool, int, string memory) {
```

Comment:
nit: can we add a quick comment above the function denoting what the return values are? Since there are 3, it's a little difficult to tell what their purpose is.

Reply:
sure and done.